queue/abstract.lua

```diff
@@ -523,11 +535,11 @@ function method.start()
 
     local _cons = box.space._queue_consumers
     if _cons == nil then
-        -- session, fid, tube, time
+        -- connection, fid, tube, time
         _cons = box.schema.create_space('_queue_consumers', {
             temporary = true,
             format = {
-                {name = 'session_id', type = num_type()},
+                {name = 'connection_id', type = num_type()},
```

Comment:
I'm not sure we can do that. Have you tried to upgrade queue created before this patch and after?

After format won't changed because `box.space._queue_consumers` is not nil.
```lua
tarantool> box.schema.space.create('tmp', {temporary = true})
---
- engine: memtx
  before_replace: 'function: 0x0114821f38'
  field_count: 0
  is_sync: false
  ck_constraint: []
  on_replace: 'function: 0x0114821f00'
  temporary: true
  index: []
  is_local: false
  enabled: false
  name: tmp
  id: 539
- created
...

tarantool> box.space.tmp
---
- engine: memtx
  before_replace: 'function: 0x0114821f38'
  field_count: 0
  is_sync: false
  ck_constraint: []
  on_replace: 'function: 0x0114821f00'
  temporary: true
  index: []
  is_local: false
  enabled: false
  name: tmp
  id: 539
...
-- restart instance here. It will be cleared by physically it still exists
box.space.tmp
---
- engine: memtx
  before_replace: 'function: 0x01084723e0'
  field_count: 0
  is_sync: false
  ck_constraint: []
  on_replace: 'function: 0x0108472220'
  temporary: true
  index: []
  is_local: false
  enabled: false
  name: tmp
  id: 539
...

```

Reply:
The "name" is not used inside the queue module.